common/components/whats-new-banner/template.njk

```diff
@@ -1,4 +1,4 @@
-{% if not params.error %}
+{% if params.title and params.date and params.body %}
```

Comment:
I'm wondering whether we need this now and instead should modify where it's called:

https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/07b7dedbbeb3f67a01527a18e4a7f1195b9ba322/common/templates/layouts/base.njk#L154-L156

To include the check for the `whatsNewContent`:

```
  {% if FEATURE_FLAGS.WHATS_NEW_BANNER and REQUEST_PATH === '/' and whatsNewContent %}
    {{ appWhatsNewBanner(whatsNewContent.bannerContent) }}
  {% endif %}
```

Because I'm wondering what happens right now when `whatsNewContent` is set to `null` (either because there are no entries or because Contentful is down) and the template tries to do `whatsNewContent.bannerContent`, would that cause the entire dashboard to fail to render?

Reply:
Adding to this, it feels better to have the conditional outside of the template too.